dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/SdkInternalLoggerTest.kt

```diff
@@ -444,6 +445,51 @@ internal class SdkInternalLoggerTest {
             )
     }
 
+    @Test
+    fun `𝕄 send metric 𝕎 metric`(
+        @StringForgery fakeMessage: String,
+        forge: Forge
+    ) {
+        // Given
+        val mockRumFeatureScope = mock<FeatureScope>()
+        whenever(mockSdkCore.getFeature(Feature.RUM_FEATURE_NAME)) doReturn mockRumFeatureScope
+        val fakeAdditionalProperties = forge.exhaustiveAttributes()
+        val mockLambda: () -> String = mock()
+        whenever(mockLambda.invoke()) doReturn fakeMessage
+        // When
+        testedInternalLogger.metric(
+            mockLambda,
+            fakeAdditionalProperties
+        )
+
+        // Then
+        verify(mockRumFeatureScope)
+            .sendEvent(
+                mapOf(
+                    "type" to "mobile_metric",
+                    "message" to fakeMessage,
+                    "additionalProperties" to fakeAdditionalProperties
+                )
+            )
+    }
+
+    @Test
+    fun `𝕄 do nothing metric 𝕎 metric { rum feature not initialized }`(
+        @StringForgery fakeMessage: String,
+        forge: Forge
+    ) {
+        // Given
+        whenever(mockSdkCore.getFeature(Feature.RUM_FEATURE_NAME)) doReturn null
+        val fakeAdditionalProperties = forge.exhaustiveAttributes()
+        val mockLambda: () -> String = mock()
+        whenever(mockLambda.invoke()) doReturn fakeMessage
+        // When
+        testedInternalLogger.metric(
+            mockLambda,
+            fakeAdditionalProperties
+        )
```

Comment:
What is the point of the test without verification part?

Reply:
I cannot verify anything there, I wanted to make sure is not throwing anything...maybe there is a method to assert that `isNotThrowing`

Reply:
yup, that's what I also have in mind ;)